__tests__/util/index.js

```diff
@@ -49,167 +41,115 @@ function mkdtempSync () {
+    { cwd: prototypePath, env: { ...process.env, env: 'test' }, stdio: 'inherit' }
+  )
 }
 
-/**
- * Create a test prototype from the current release archive
- *
- * Creates a prototype at `prototypePath`.
- *
- * @param {string} prototypePath
- * @param {Object} [options]
- * @param {string} [options.archivePath] - Path to archive to use to create prototype, if not provided uses mkReleaseArchive
- * @returns {void}
- */
-async function mkPrototype (prototypePath, { archivePath, overwrite = false } = {}) {
-  if (!overwrite && fs.existsSync(prototypePath)) {
-    const err = new Error(`path already exists '${prototypePath}'`)
-    err.path = prototypePath
-    err.code = 'EEXIST'
-    throw err
-  }
-
-  archivePath = archivePath || await mkReleaseArchive()
-
-  await fs.promises.mkdir(prototypePath, { recursive: true })
-
-  await tar.extract({ cwd: prototypePath, file: archivePath, strip: 1 })
+function startPrototype (prototypePath) {
+  child_process.execSync(
+    'npm start',
+    { cwd: prototypePath, env: { ...process.env, env: 'test' }, stdio: 'inherit' }
+  )
 }
 
-/**
- * Synchronous version of `mkPrototype()`
- *
- * See the note in the docstring for `mkReleaseArchive()` for a warning against
- * using this function. Prefer the async version where possible.
- *
- * @param {string} prototypePath
- * @param {Object} [options]
- * @param {string} [options.archivePath] - Path to archive to use to create prototype, if not provided uses mkReleaseArchiveSync
- * @returns {void}
- */
-function mkPrototypeSync (prototypePath, { archivePath, overwrite = false } = {}) {
-  if (!overwrite && fs.existsSync(prototypePath)) {
-    const err = new Error(`path already exists '${prototypePath}'`)
-    err.path = prototypePath
-    err.code = 'EEXIST'
-    throw err
-  }
-
-  archivePath = archivePath || mkReleaseArchiveSync()
-
-  fs.mkdirSync(prototypePath, { recursive: true })
-
-  tar.extract({ cwd: prototypePath, file: archivePath, strip: 1, sync: true })
-}
+// /**
+//  * Synchronous version of `mkPrototype()`
+//  *
+//  * See the note in the docstring for `mkReleaseArchive()` for a warning against
+//  * using this function. Prefer the async version where possible.
+//  *
+//  * @param {string} prototypePath
+//  * @param {Object} [options]
+//  * @param {string} [options.archivePath] - Path to archive to use to create prototype, if not provided uses mkReleaseArchiveSync
+//  * @returns {void}
+//  */
+// function mkPrototypeSync (prototypePath, { archivePath, overwrite = false } = {}) {
+//   if (!overwrite && fs.existsSync(prototypePath)) {
+//     const err = new Error(`path already exists '${prototypePath}'`)
+//     err.path = prototypePath
+//     err.code = 'EEXIST'
+//     throw err
+//   }
+//
+//   archivePath = archivePath || mkReleaseArchiveSync()
+//
+//   fs.mkdirSync(prototypePath, { recursive: true })
+//
+//   tar.extract({ cwd: prototypePath, file: archivePath, strip: 1, sync: true })
+// }
```

Comment:
Might be simpler to use multi line comment to comment out this block if we don't want to delete it.